managed/services/backup/pitr_timerange_service.go

```diff
@@ -54,8 +54,8 @@ type oplogChunk struct {
 	RS          string              `bson:"rs"`
 	FName       string              `bson:"fname"`
 	Compression compressionType     `bson:"compression"`
-	StartTS     primitive.Timestamp `bson:"start_ts"`
-	EndTS       primitive.Timestamp `bson:"end_ts"`
+	StartTS     primitive.Timestamp `bson:"startTs"`
+	EndTS       primitive.Timestamp `bson:"endTs"`
```

Comment:
I think we need to revert it and comment to ignore it since it's based on data that we get from MongoDB.